src/site/content/en/fast/font-best-practices/index.md

```diff
@@ -189,90 +129,59 @@ serve stylesheets and fonts from separate origins. For example, this is how the
 </head>
 ```
 
-
 {% Aside %}
-
-[Google Fonts](https://fonts.google.com/) provides the option to load fonts via
-`<link>` tags or an `@import` statement. The `<link>` code snippet includes a
-`preconnect` resource hint and therefore will likely result in faster stylesheet
-delivery than using `@import` version. These `<link>` tags should be placed as
-early in the document as possible.
-
-
+  [Google Fonts](https://fonts.google.com/) provides the option to load fonts via `<link>` tags or an `@import` statement. The `<link>` code snippet includes a `preconnect` resource hint and therefore will likely result in faster stylesheet
+  delivery than using `@import` version. These `<link>` tags should be placed as early in the document as possible.
 {% endAside %}
 
+### Be cautious when using `preload` to load fonts
 
-#### Avoid using preload to load fonts
+Although `preload` is highly effective at making fonts discoverable early in the page load process, this comes at the cost of taking away browser resources from the loading of other resources.
 
-Generally speaking, using the
-[`preload`](https://developer.mozilla.org/docs/Web/HTML/Preloading_content)
-resource hint to load fonts should be avoided. Although `preload` is highly
-effective at making fonts discoverable early in the page load process, this
-comes at the cost of taking away browser resources from the loading of other
-resources.
-
-In most scenarios, inlining font declarations and adjusting stylesheets is
-a more effective approach. These adjustments come closer to addressing the root
-cause of late-discovered fonts - rather than just providing a workaround.
-
-In addition, using `preload` as a font-loading strategy should also be used
-carefully as it bypasses some of the browser's built-in content negotiation
-strategies. For example, `preload` ignores `unicode-range` declarations, and if
-used prudently, should only be used to load a single font format.
+Inlining font declarations and adjusting stylesheets may be a more effective approach. These adjustments come closer to addressing the root cause of late-discovered fonts—rather than just providing a workaround.
 
+In addition, using `preload` as a font-loading strategy should also be used carefully as it bypasses some of the browser's built-in content negotiation strategies. For example, `preload` ignores `unicode-range` declarations, and if used prudently, should only be used to load a single font format.
```

Comment:
This feels like it should be in an `'important'` aside. I didn't know this!

Reply:
I'm not sure how much it matters really. You would preload the most likely used font (which can be a subsetted font, later referenced in a font-face). I considered removing this completely because of that. So don't think we should add an `'important'`.